common/src/main/java/com/microsoft/identity/common/internal/request/MsalBrokerRequestAdapter.java

```diff
@@ -257,6 +265,44 @@ public BrokerAcquireTokenSilentOperationParameters brokerSilentParametersFromBun
         return new HashSet<>(Arrays.asList(scopes));
     }
 
+    public static Bundle getSilentBrokerRequestBundle(final AcquireTokenSilentOperationParameters parameters) {
+        final MsalBrokerRequestAdapter msalBrokerRequestAdapter = new MsalBrokerRequestAdapter();
+
+        final Bundle requestBundle = new Bundle();
+        final BrokerRequest brokerRequest = msalBrokerRequestAdapter.
+                brokerRequestFromSilentOperationParameters(parameters);
+
+        requestBundle.putString(
+                AuthenticationConstants.Broker.BROKER_REQUEST_V2,
+                new Gson().toJson(brokerRequest, BrokerRequest.class)
+        );
+
+        requestBundle.putInt(
+                AuthenticationConstants.Broker.CALLER_INFO_UID,
+                parameters.getAppContext().getApplicationInfo().uid
+        );
+
+        return requestBundle;
+    }
+
+    public static Bundle getRequestBundleForRemoveAccount(@NonNull final OperationParameters parameters) {
+        final Bundle requestBundle = new Bundle();
+        if (null != parameters.getAccount()) {
+            requestBundle.putString(ENVIRONMENT, parameters.getAccount().getEnvironment());
+            requestBundle.putString(ACCOUNT_HOME_ACCOUNT_ID, parameters.getAccount().getHomeAccountId());
+        }
+
+        return requestBundle;
+    }
+
+    public static Bundle getRequestBundleForGetAccounts(@NonNull final OperationParameters parameters) {
+        final Bundle requestBundle = new Bundle();
+        requestBundle.putString(ACCOUNT_CLIENTID_KEY, parameters.getClientId());
```

Comment:
Can we rename to `ACCOUNT_CLIENTID_KEY` to `ACCOUNT_CLIENT_ID_KEY`?

Reply:
I'll keep it as is right now to keep the amount of change as minimal as possible - MSAL and Broker change is already big enough IMHO.